### ISSUE TITLE
Pie in stock with accessibility- error in the console caused by navigator

### ISSUE THREAD

**Issue by karolkolodziej** [highcharts/highcharts#16085]
#### Expected behaviour
No errors in the console.

#### Actual behaviour
When the accessibility module is loaded, hovering over the pie series with `stockChart` constructor generates an error in the console.
Without a navigator works fine.

#### Live demo with steps to reproduce
http://jsfiddle.net/BlackLabel/yx54bkq9/

Without navigator: http://jsfiddle.net/BlackLabel/f1zm87hc/

#### Product version
Highstock JS v9.1.2 (2021-06-16) on branch `bugfix/13868-hovering-pie-in-stock` which fixes some issues 

#### Affected browser(s)
All
